virus/p_intestivirus

```diff
@@ -0,0 +1,84 @@
+### {A Proposed Microbiota Stool Bus as a Mode of Infection for a Novel Posa-like Virus}
+written by: [Derek Tea](https://github.com/DerekTea)
+
+[Cover art Intestivirus](virus/img/p_intestivirus/cover_art_intestivirus.jfif)
```

Comment:
Image paths should be `(img/p_intesti...)` The file is already in the `virus/` path and so you're telling it to look for the image in `virus/virus/img/` which does not exist.